components/02-elements/mega-menu/mega-menu.hbs

```diff
@@ -0,0 +1,55 @@
+<div class="mega-menu">
+    <ul class="mega-menu__wrapper container">
+        {{#each list}}
+            <li class="mega-menu__item {{ class }}">
+                <a href="#" class="mega-menu__link">
```

Comment:
this `href` value should be set in config? B/c this item should have some link, not be an empty link?

Reply:
added to config.